README.md

```diff
@@ -64,6 +66,11 @@ ReactDom.render(
       appendTo: 'head', // optional, default to "head", can be "head" or "body",
       nonce: undefined // optional, default undefined
     }}
+    inlineBadgeId="g-recaptcha" // optional to make it inline (there must exists a container with this Id)
+    parameters={{// optional inline badge configuration
```

Comment:
Since this property only works with inlineBage, I suggest making a property: "inlineBadge":

```ts
inlineBadge={{ id: 'g-recaptcha', parameters: { theme: 'dark' } }}
```